### PR TITLE
feature/comment

### PR DESCRIPTION
**Pull request by igaryakqwe**

### CODE REVIEW

File: src/components/common/ui/comment/Comment.styles.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const date: SxProps<Theme> = {
+  height: '17px',
+  width: '100%',
+  fontFamily: 'Manrope',
+  fontStyle: 'normal',
+  fontWeight: 400,
+  fontSize: '14px',
+  lineHeight: '140%',
+  textAlign: 'right',
+  color: '#737373'
```
Comment (kujo205) [inline src/components/common/ui/comment/Comment.styles.ts]: use color from the theme

File: src/components/common/ui/comment/Comment.styles.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  maxWidth: '740px',
+  height: '100%',
+  padding: '11px 16px 12px 16px',
+  backgroundColor: '#292929',
+  borderRadius: '4px',
+  margin: '10px'
+}
+
+const text: SxProps<Theme> = {
+  fontFamily: 'Manrope',
```
Comment (kujo205) [inline src/components/common/ui/comment/Comment.styles.ts:13]: use font styles from the theme as well

File: src/components/common/ui/comment/Comment.styles.ts [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import {SxProps, Theme} from "@mui/material/styles";
+
+const wrapper: SxProps<Theme> = {
+  maxWidth: '740px',
+  height: '100%',
+  padding: '11px 16px 12px 16px',
+  backgroundColor: '#292929',
```
Comment (kujo205) [inline src/components/common/ui/comment/Comment.styles.ts:7]: same with color

File: src/components/common/ui/comment/Comment.tsx [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import {Box, Typography} from "@mui/material";
+import * as styles from "./Comment.styles";
+
+const Comment = ({text, semester, year}) => {
```
Comment (kujo205) [inline src/components/common/ui/comment/Comment.tsx]: Add interface for better types support, like that

`interface CommentProps{
text:string;
semester:string;
year:string;
}`

And u can add this interface as shown [here](https://github.com/fictadvisor/fictadvisor-web/blob/34c222fef0de24db83add29f16998e654d55f936/src/components/common/ui/alert-mui/Alert.tsx#LL39C25-L39C25)

File: src/components/common/ui/comment/Comment.styles.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  padding: '11px 16px 12px 16px',
+  backgroundColor: 'backgroundDark.200',
+  borderRadius: '4px',
+};
+
+const text: SxProps<Theme> = {
+  marginBottom: '2px',
+  textAlign: 'justify',
+  typography: {
+    mobile: typography.body1,
```
Comment (Markusplay) [inline src/components/common/ui/comment/Comment.styles.ts:17]: Like in a `backgroundColor`, you can write just `body1`. Same for other typographies

File: src/components/pages/test-pages/comment-page/CommentTestPage.styles.ts [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+const wrapper: SxProps<Theme> = {
+  width: '100%',
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+  gap: '10px',
+};
+
+export { wrapper };
```
Comment (Markusplay) [inline src/components/pages/test-pages/comment-page/CommentTestPage.styles.ts:12]: Actually, it is not necessary to write export like it. You can just write `export`  before `const`, result is the same. Notice, that If you would have huge amount of styles, you will write looooong export.
P.S : same issue in Comment.styles

**Review by Markusplay — CHANGES_REQUESTED** [review]

**Review by Markusplay — APPROVED** [review]

**Pull request merged**